other/text_to_speech.py

```diff
@@ -0,0 +1,116 @@
+"""
+Created by: Ariel Sperduti
+
+Given a text it converts into an  MP3 audio file.
+Under the hood it uses fromtexttospeech.com.
+
+Example:
+    text_to_speech(text="¡Hola Mundo!", language="Spanish", voice="Mateo")
+"""
+
+import re
+
+import requests
+
+LANGUAGES_VOICES = {
+    "US English": ["Alice", "Daisy", "George", "Jenna", "John"],
+    "British": ["Emma", "Harry"],
+    "French": ["Gabriel", "Jade"],
+    "Spanish": ["Isabella", "Mateo"],
+    "German": ["Michael", "Nadine"],
+    "Italian": ["Alessandra", "Giovanni"],
+    "Portuguese": ["Rodrigo"],
+    "Russian": ["Valentina"],
+}
+
+VOICES = {
+    "Emma": "IVONA Amy22 (UK English)",
+    "Harry": "IVONA Brian22 (UK English)",
+    "Jade": "IVONA CΘline22 (French)",
+    "Gabriel": "IVONA Mathieu22 (French)",
+    "Nadine": "IVONA Marlene22 (German)",
+    "Michael": "IVONA Hans22 (German)",
+    "Valentina": "IVONA Tatyana22 (Russian)",
+    "John": "IVONA Eric22",
+    "Jenna": "IVONA Jennifer22",
+    "George": "IVONA Joey22",
+    "Alice": "IVONA Kimberly22",
+    "Daisy": "IVONA Salli22",
+    "Alessandra": "IVONA Carla22 (Italian)",
+    "Giovanni": "IVONA Giorgio22 (Italian)",
+    "Isabella": "IVONA Conchita22 (Spanish [Modern])",
+    "Mateo": "IVONA Enrique22 (Spanish [Modern])",
+    "Rodrigo": "IVONA Cristiano22 (Portuguese)",
+}
+
+
+def get_mp3_filename(html: str) -> str:
```

Comment:
```suggestion
def get_mp3_filename(html: str) -> Optional[str]:
```
As you're returning `None` or `str`.